src/PeclEvLoop.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+namespace React\EventLoop;
+
+use Ev;
+use EvLoop;
+use React\EventLoop\Tick\FutureTickQueue;
+use React\EventLoop\Timer\Timer;
+use React\EventLoop\Timer\TimerInterface;
+use SplObjectStorage;
+
+/**
+ * @see https://bitbucket.org/osmanov/pecl-ev/overview
+ */
+class PeclEvLoop implements LoopInterface
+{
+    private $loop;
+    private $futureTickQueue;
+    private $timers;
+    private $readStreams = [];
+    private $writeStreams = [];
+    private $running;
+
+    public function __construct()
+    {
+        $this->loop             = new EvLoop();
+        $this->futureTickQueue  = new FutureTickQueue($this);
+        $this->timers           = new SplObjectStorage();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addReadStream($stream, callable $listener)
```

Comment:
@jsor, no problem, I can remove it, but `{@inheritdoc}` makes inheritance explicit ([link to PSR-5 inheritance section](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#61-making-inheritance-explicit-using-the-inheritdoc-tag)). Is it a `reactphp` project convention?